pkg/tests/make_rpm_test.py

```diff
@@ -156,13 +157,14 @@ def testBuild(self):
           '#!/bin/sh',
           'mkdir -p RPMS',
           'touch RPMS/test.rpm',
+          'echo "$@" > RPMS/test.rpm',
           'echo "Wrote: $PWD/RPMS/test.rpm"',
       )
       os.chmod(dummy, 0o777)
 
       with PrependPath([outer]):
         # Create the builder and exercise it.
-        builder = make_rpm.RpmBuilder('test', '1.0', '0', 'x86', None)
+        builder = make_rpm.RpmBuilder('test', '1.0', '0', 'x86_64-linux', None)
```

Comment:
The target should conform to the format we say we require. Maybe  'x86_64-generic-linux'